### DISCUSSION TITLE
uncategorized/RK3588%E8%BD%AC%E6%8D%A2%E6%A8%A1%E5%9E%8B%E6%B5%81%E7%A8%8B/

### DISCUSSION THREAD

**Discussion by giscus** [APLaS-Plus/APLaS-Plus.github.io#6] [Announcements]
# uncategorized/RK3588%E8%BD%AC%E6%8D%A2%E6%A8%A1%E5%9E%8B%E6%B5%81%E7%A8%8B/

简介本文章介绍了怎么在搭载了RK3588芯片的主机下使用YOLO系列模型，需要下载的必要资源库ultralytics ，rknn_model_zoo ，rknn-toolkit2 ，具体的流程为：   本例程建立在你了解Python虚拟环境的条件下  作者实验环境 瑞芯微主板环境：众云世纪 RK3588 高性能人工智能主板 ZYSJ-2288A pt –> onnx 环境：x86 win11

https://aplas-plus.github.io/uncategorized/RK3588%E8%BD%AC%E6%8D%A2%E6%A8%A1%E5%9E%8B%E6%B5%81%E7%A8%8B/

**Comment by APLaS-Plus** [comment]
哇~这博主写的也太好了吧😍
